src/lib/code-sample/schema.ts

```diff
@@ -63,21 +71,36 @@ export const createCodeSample = (
         title: 'JavaScript',
         request: createJavascriptRequest(codeSampleDefinition, context),
         response: createJavascriptResponse(codeSampleDefinition, context),
+        request_syntax: 'javascript',
+        response_syntax: 'json',
       },
       python: {
         title: 'Python',
         request: createPythonRequest(codeSampleDefinition, context),
         response: createPythonResponse(codeSampleDefinition, context),
+        request_syntax: 'python',
+        response_syntax: 'python',
       },
       ruby: {
         title: 'Ruby',
         request: createRubyRequest(codeSampleDefinition, context),
         response: createRubyResponse(codeSampleDefinition, context),
+        request_syntax: 'ruby',
+        response_syntax: 'ruby',
       },
       php: {
         title: 'PHP',
         request: createPhpRequest(codeSampleDefinition, context),
         response: createPhpResponse(codeSampleDefinition, context),
+        request_syntax: 'php',
+        response_syntax: 'json',
```

Comment:
Not for this PR, but this really should be php

Reply:
https://github.com/seamapi/blueprint/issues/60